websites/L/Los Angeles Times/presence.ts

```diff
@@ -0,0 +1,202 @@
+const presence = new Presence({
+		clientId: "1005501034191409153",
+	}),
+	time = Math.floor(Date.now() / 1000);
```

Comment:
Just for repo consistency
```suggestion
	browsingTimestamp = Math.floor(Date.now() / 1000);
```